### ISSUE TITLE
Also extract Bookmarks?

### ISSUE THREAD

**Issue by dufferzafar** [kcp18/browserhistory#3]
Hey, cool project!

I was wondering if extracting Bookmarks is out of scope for this? Since the project is named `browserhistory`.

You already seem to have 90% of the code for this, all that is needed now is a SQL query to extract the bookmarks!

**Comment by kcp18** [comment]
Hello dufferzafar!
  That sounds like a plan. I'll add the bookmark feature to the module by the end of this week or later.

**Comment by dufferzafar** [comment]
Will you also be renaming the project then? :stuck_out_tongue_winking_eye:

**Comment by kcp18** [comment]
I'll think about it. If you have any suggestions, let me know! Thank you for your feedback.

**Comment by dufferzafar** [comment]
Yeah, I feel you should rename the project. Perhaps to something generic like `browserdata` (which is available on PyPI - in case you want to push your package there.)

**Comment by kcp18** [comment]
That's a generic and simple name for a package, what a name! @dufferzafar, would you like to participate in the project? If you're interested, shoot me an email at parkchan@brandeis.edu

**Comment by dufferzafar** [comment]
Yeah, sure!
